src/invidious/routes/preferences.cr

```diff
@@ -1,5 +1,5 @@
-class Invidious::Routes::PreferencesRoute < Invidious::Routes::BaseRoute
-  def show(env)
+module Invidious::Routes::PreferencesRoute
```

Comment:
To be consistant with the others, could you rename this one to `Invidious::Routes::Preferences`? `PreferencesRoute` is kinda redundant

Reply:
That'll cause a name collision with the global Preferences constant.

Reply:
There is no such constant, only the `Preferences` class, but as those are in different namespaces, and will be rewritten soon, that won't be a problem!

Reply:
> There is no such constant, only the `Preferences struct`

Ah that's what I meant thanks!

> those are in different namespaces, and will be rewritten soon, that won't be a problem!

Different namespaces yes, but the `Preferences` struct is global and gets overwritten by the module instead in that scope. So, if I go ahead and rename the `PreferencesRoute` module to `Preferences` then Crystal would think I'm attempting to call the `#from_json` method of the module here:

https://github.com/iv-org/invidious/blob/048eef100d330a83100522e85e9425d2aa921057/src/invidious/routes/preferences.cr#L132-L163 

Instead of the the `Preferences` struct

Reply:
Ah, mmh, indeed. we'll take care of that once I've finished my user preferences rewrite PR!